src/main/resources/db/changelog/changeset/13-create-cartitem-table.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-cartitem-table
+      author: Vladyslav_Vyshynskyi
+      changes:
+        - createTable:
+            tableName: cartitems
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: shoppingcart_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: quantity
+                  type: int
+                  constraints:
+                    nullable: false
+            foreignKeyConstraints:
+              - foreignKeyConstraint:
+                  baseColumnNames: shoppingcart_id
+                  baseTableName: cartitems
+                  referencedColumnNames: id
+                  referencedTableName: shoppingcarts

```

Comment:
let's add a constraint name, becasue I think we might have an error because of that, same for other relation